gammapy/utils/fitting/fit.py

```diff
@@ -332,22 +338,33 @@ def likelihood_contour(self):
         """
         raise NotImplementedError
 
-    def minos_contour(self):
+    def minos_contour(self, x, y, numpoints=10, sigma=1.0):
         """Compute MINOS contour.
 
         This is a contouring algorithm for a 2D function
         which is not simply the likelihood function.
-        That 2D function is given at each point (par_1, par_2)
+        That 2D function is given at each point ``(par_1, par_2)``
         by re-optimising all other free parameters,
         and taking the likelihood at that point.
 
         Very compute-intensive and slow.
 
         Calls ``iminuit.Minuit.mncontour``
-        TODO: what does Sherpa do?
-        TODO: find a better name.
         """
-        raise NotImplementedError
+        parameters = self._parameters
+        x = parameters[x]
+        y = parameters[y]
+
+        with parameters.restore_values:
+            result = mncontour(self.minuit, parameters, x, y, numpoints, sigma)
+
+        return {
+            "is_valid": result["x_info"]["is_valid"] and result["y_info"]["is_valid"],
```

Comment:
I think for the other methods such as `.optimize(), `.covariance()` and `confidence()` we called the status flac `success`. Maybe make it consistent?

Reply:
will do